components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.html

```diff
@@ -0,0 +1,92 @@
+<chef-modal [visible]="visible" (closeModal)="closeCreateModal()">
```

Comment:
This template, code, scss, and tests, were essentially bulk moved from UserManagementComponent and UserFormComponent, so the behavior should be unchanged.